cmd/node/factory/structs.go

```diff
@@ -918,6 +918,7 @@ func generateGenesisHeadersAndApplyInitialBalances(args *processComponentsFactor
 		TrieStorageManagers:      args.tries.TrieStorageManagers,
```

Comment:
@LucianMincu did some testing with this branch ans we find out that providing an invalid pub key returns the OK status. Please check auction.go, getVerifiedBLSKeysFromArgs function, L475. I think we should escalate the error in order to stop the staking process.

Reply:
fixes on the way.